### PR TITLE
Update APICAST_HTTPS_VERIFY_DEPTH description

### PR DESCRIPTION
**Pull request by pehala**
Depth 1 should mean `Root CA -> Certificate`, Depth 2 should be `Root CA -> Intermediate CA -> Certificate` and so on.

### CODE REVIEW

File: doc/parameters.md [hunk 1]
```diff
@@ -408,7 +408,7 @@ Path to a file with the X.509 certificate secret key in the PEM format.
 **Values:** positive integers
 
 Defines the maximum length of the client certificate chain.
-If this parameter has 1 as its value, it implies that this length might include one additional certificate (eg. intermediate CA).
+If this parameter has 1 as its value, it implies that this length might include one additional certificate (eg. root CA).
```
Comment (porueesq) [inline doc/parameters.md:411]:
```suggestion
If this parameter has `1` as its value, it implies that this parameter includes an additional certificate. For example,  root CA.
```

**Review by porueesq — CHANGES_REQUESTED** [review]
A suggestion

File: doc/parameters.md (hunk 1 above)
Comment (pehala) [reply doc/parameters.md]: Thanks, I changed it based on your suggestion.

**Review by porueesq — APPROVED** [review]
image

**Pull request merged**